src/app/containers/Timestamp/index.jsx

```diff
@@ -80,10 +26,19 @@ const TimestampContainer = ({ lastPublished, firstPublished }) => {
     return null;
   }
 
+  const publishedDatetime = formatUnixTimestamp(firstPublished, longNumeric);
+  const updatedDatetime = formatUnixTimestamp(lastPublished, longNumeric);
+
   return (
     <GridItemConstrainedMedium>
-      {defaultTimestamp(firstPublished)}
-      {updatedTimestamp(lastPublished, firstPublished)}
+      <Timestamp datetime={publishedDatetime}>
+        {humanReadable(firstPublished)}
```

Comment:
Just noticed this, have the requirements changed? i thought the 'first published' date shouldn't be relative?